external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -50,6 +50,9 @@
     <PackageReference Include="Microsoft.3rdpartytools.MarkdownLog">
       <Version>0.10.0-alpha-experimental</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions">
+      <Version>2.1.0-preview1-25806-02</Version>
```

Comment:
It would be nice to either reuse this property:

https://github.com/dotnet/corefx/blob/e273e7f2bebfe7681c33da106be086a6ae534a19/dependencies.props#L34

Or even better, define a new `MicrosoftDotNetPlatformAbstractionsPackageVersion` property in dependencies.props.  That way this version automatically gets updated when a new version comes out.